harness/determined/common/api/bindings.py

```diff
@@ -6653,52 +6653,33 @@ def __init__(
         *,
         closeTrial: "typing.Optional[v1CloseTrialOperation]" = None,
         createTrial: "typing.Optional[v1CreateTrialOperation]" = None,
-        searcherProgress: "typing.Optional[v1SearcherProgressOperation]" = None,
-        shutdown: "typing.Optional[v1ShutdownOperation]" = None,
+        setSearcherProgress: "typing.Optional[v1SetSearcherProgressOperation]" = None,
+        shutDown: "typing.Optional[v1ShutDownOperation]" = None,
         validateAfter: "typing.Optional[v1ValidateAfterOperation]" = None,
     ):
         self.validateAfter = validateAfter
         self.createTrial = createTrial
         self.closeTrial = closeTrial
-        self.shutdown = shutdown
-        self.searcherProgress = searcherProgress
+        self.shutDown = shutDown
```

Comment:
Come to think of it, I agree. This is a verb, not a noun.  https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/shut-down-shutdown